Lesson2/src/main/java/pl/slvd/university/Applicant.java

```diff
@@ -0,0 +1,84 @@
+package pl.slvd.university;
+
+import java.util.*;
+
+import static java.util.Arrays.stream;
+
+import static pl.slvd.university.ExamSheet.grades;
+
+public class Applicant {
+    private final short id;
+    private final String firstLastName;
+    private final String dateOfBirth;
+    private final String faculty;
```

Comment:
faculty and speciality can be enums, so to reduce the number of possible mistakes from user side and unify data across your apllication.

Reply:
It can be a variable of class Faculty, why do you use String here?